desktop-widgets/configuredivecomputerdialog.cpp

```diff
@@ -14,9 +14,27 @@
 #include <QProgressDialog>
 #include <QSettings>
 
+bool diveComputerConfigurable(QString vendor, QString product)
+{
+	if ((vendor == "Heinrichs Weikamp" &&
+	     product != "OSTC") ||
```

Comment:
As far as I've know, the original "OSTC" runs the same firmware as the mk2, Thus it works just fine to configure it.